client/src/containers/Organisation/Organisation.js

```diff
@@ -0,0 +1,99 @@
+import React, { Component } from 'react';
+import { Container, Jumbotron, Button } from 'reactstrap';
+import { EventList } from '../../components/EventList/EventList';
+import { fetchOrganisationDetail } from '../../actions';
+import { connect } from 'react-redux';
+
+type Props = BaseReduxPropTypes & {
+  organisation: Object,
+};
+
+class Organisation extends Component<Props> {
+  constructor(props) {
+    super(props);
+    this.state = { organisation: {} };
+  }
+  componentDidMount() {
+    this.getOrganisationDetail();
+    const mockOrganisation = {
```

Comment:
@QaseemLodhi why are we still using mock data?

Reply:
some field were not in api object but I will resolve it asap.